scripts/benchmark.ts

```diff
@@ -10,6 +10,7 @@ const txPerCommitment = 32;
 const commitmentsPerBatch = 32;
 const blockTime = 13;
 const blockGasLimit = 12500000;
+const batchID = 1;
```

Comment:
Should this be a different value?

Reply:
batchID=1 makes sense to me since batchID 0 is the genesis.